### DISCUSSION TITLE
Open Files App/Open Folder buttons do nothing, “Unsupported folder” when trying to add external folder on iOS 17

### DISCUSSION THREAD

**Discussion by greyivy** [MobiusSync/MobiusSync#162] [Q&A]
I’m not able to access any files synced by this app, sandboxed or otherwise. 

iOS 17.3.1

**Comment by Radieserlbrot** [comment]
I am on iOS 17.4 and also have this problem with an external folder. Can anyone help how to circumvent this issue?
Thanks a lot

**Comment by kahilah** [comment]
Also experiencing this when trying to choose a folder from icloud section of directory structure. OneDrive is greyed out which I presume is due to App restrictions?

**Comment by SimonPickup** [comment]
Mobius Sync only supports external folders under "On My iPhone", not files under other file provider locations like iCloud Drive or Dropbox etc.

If you are having problems with other folders, please try to reproduce on the latest TestFlight beta (see #156) and raise as a new issue.

**Reply by dadaphl** [reply]
Syncing iCloud drive folder was the reason I purchased the full version to begin with. only to find out after purchase that it doesn't work. The website is also pretty shy about it. 
A modal that asked me to purchase exactly when I selected the iCloud folder popped. I happily complied to. Exactly after finishing the purchase it showed me the "Unsupported folder" error. I'm sure this is not intended, It's a great product and I appreciate the effort that went into it, but I could be starting to feel a little bamboozled at this point. I think it would be good to have that information in the FAQ on the website. And/or show unsupported folder error before the purchase modal.
